src/Perpetuum.RequestHandlers/Zone/Containers/ZoneChangeModule.cs

```diff
@@ -58,14 +58,14 @@ public override void HandleRequest(IZoneRequest request)
                 player.Armor = player.ArmorMax * hpRatio;
                 player.Core = player.CoreMax * apRatio;
 
+
```

Comment:
Why the extra line break? If you're wanting extra segregation, you're probably better creating methods and moving it that way. SRP after all...